tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -1,27 +1,25 @@
 #!/bin/bash
 set -e
-#install protobuf
-mkdir -p /tmp/src
-mkdir -p /opt/cmake
-aria2c -q -d /tmp/src https://github.com/Kitware/CMake/releases/download/v3.13.2/cmake-3.13.2-Linux-x86_64.tar.gz
-tar -xf /tmp/src/cmake-3.13.2-Linux-x86_64.tar.gz --strip 1 -C /opt/cmake
-aria2c -q -d /tmp/src https://github.com/protocolbuffers/protobuf/archive/v3.6.1.tar.gz
-tar -xf /tmp/src/protobuf-3.6.1.tar.gz -C /tmp/src
-cd /tmp/src/protobuf-3.6.1
-if [ -f /etc/redhat-release ] ; then
-  PB_LIBDIR=lib64
-else
-  PB_LIBDIR=lib
+
+SYS_LONG_BIT=$(getconf LONG_BIT)
+
+echo "Installing azcopy"
+if [ $SYS_LONG_BIT = "64" ]; then
+  mkdir -p /tmp/azcopy
+  aria2c -q -d /tmp/azcopy -o azcopy.tar.gz https://aka.ms/downloadazcopy-v10-linux
+  tar --strip 1 -xf /tmp/azcopy/azcopy.tar.gz -C /tmp/azcopy
+  cp /tmp/azcopy/azcopy /usr/bin
 fi
```

Comment:
Why download azcopy only for 64 bit? Seems confusing.

Reply:
Because it doesn't have a 32 bit version